docs/download/downloading_data.md

```diff
@@ -0,0 +1,261 @@
+
+    ![Bionumbus-Page](images/bionimbus-page.png)
+
+4. This will redirect you to the login page for NIH iTrust.
+
+    ![NIH-iTrust-login](images/NIH-iTrust-login.png)
+
+5. After you have successfully logged in, you will be redirected back to PDC where you can create an access key. If you see `TCGA-PCAWG` in the project list and you have `download` rights, you can proceed with generating an access key. Click on `Create access key` button. A popup will show you the new access key and secret key.
+
+    ![Example-PDC-Keys](images/example-pdc-keys.png)
+
+6\. Copy these keys and add them to your awscli credentials file, which depending on your system, is usually at: `~/.aws/credentials`. The aws credentials file may look like this. Please edit it to include your own key ID and secret key
+
+```
+[pdc]
+aws_access_key_id = your_pdc_access_key_id
+aws_secret_access_key = your_pdc_secret_access_key
+```
+
+You can also run `aws configure` and follow the prompts.
+
+
+#### Download Data Instructions
+
+1. Find your data of interest in the [Data Repository](https://dcc.icgc.org/repositories) by selecting `PDC` repository. Click on `Download Files` and download manifest file.
+    ![Download-PDC-Manifest](images/download-pdc-manifest.png)
+
+2\. The manifest file that you downloaded from ICGC Data Portal for PDC is actually a shell script containing aws cli commands, one line per file. Here's an example
+```
+aws --profile pdc --endpoint-url https://bionimbus-objstore-cs.opensciencedatacloud.org s3 cp s3://pcawg-tcga-lihc-us/230e20d7-38da-5aa0-89ac-071bd509cd53 .
+```
+You can execute the above line directly on the command line, or you can execute the manifest script file to download the object(s) to local.
+
+```
+./manifest.pdc.1554823121273.sh
+```
+
+The file names of the downloaded files will be object IDs.
+
+
+## Downloading Data from the AWS repository
+
+Amazon cloud service containing ICGC data.
+
+Repository information can be found [here](https://docs.icgc.org/download/repositories/#aws)
+
+   <span style="color:blue">NOTE: In order to download data from AWS, the score-client tool must run in the same environment as the object storage system. That means you will need your own AWS account to provision a running EC2 instance and the Score Client must run on an AWS VM in the N. Virginia availability zone.</span>
+
+
+#### Prerequisites
+
+1. [Apply for DACO access](https://github.com/icgc-dcc/dcc-docs/blob/download-doc-update/docs/download/data-access.md#apply-for-access-to-controlled-data). Once you are approved by DACO, you will recieve an email from EGA about setting up your password. If you already had an EGA account from before, you will use the same username/password to access ICGC controlled data at EGA.
```

Comment:
relative URL, `[Apply for DACO access](/download/data-access/#apply-for-access-to-controlled-data)`

Reply:
Corrected with relative URL now